packages/plugin-react-query/src/templates/reactQueryCommon.liquid

```diff
@@ -39,7 +39,12 @@ Omit<FetcherOptions<
   {%- endif %}
 }
 
-export function {{fetcherName}}(props: {{fetcherPropsName}}): Promise<{{okResponseName}}> {
+export interface {{fetcherPropsName}}ReturnValue {
```

Comment:
let's come up with a better/more semantic name for this interface, avoid using `Props` keyword.. maybe `{{fetcherName}}ResponseContainer`?

Reply:
or `{{fetcherName}}ResponseHeaders `

Reply:
Renamed it to `{{typeName}}ResponseContainer` instead since the interface contains both response `content` and `headers`.

Reply:
Sample interface generated with above template:

![image](https://github.com/user-attachments/assets/f3b0a722-38f8-4c40-b81a-edb0911d2f2e)